ibm/mas_devops/roles/ocp_provision/README.md

```diff
@@ -241,6 +241,14 @@ The working directory that is used to perform the installation, it will contain
 - Environment Variable: `IPI_DIR`
 - Default Value: `~/openshift-install`
 
+### sshKey
```

Comment:
You're adding new capability, this needs to be raised as a [minor] update rather than a [patch] .. and as a result update the changelog in docs/changes.md and ibm/mas_devops/README.md

Reply:
@durera 12.2 would be the next release right ?